docs/reference/filebeat/file-identity.md

```diff
@@ -0,0 +1,183 @@
+
+## Supported options
+
+### Overview [file-identity-overview]
+
+Here is the brief high-level comparison of all currently available options:
+
+| Name | Use case | Pros | Cons |
+|------|----------|------|------|
+| path | Files are never moved or renamed, file names are never re-used. | Simple and fast. | The most unstable option, requires to maintain immutable file paths. |
+| native | Stable file systems, files < 64 bytes in size, ingestion without delays. | Low CPU / memory overhead. | Might cause data duplication or data loss if the file system provides unstable `inode` or `device` values. No support for network shares, containers or VMs.
+| inode_marker | Same as `native` but `device` number is changing. | Same as `native` + no dependency on `device` number. | Can still cause data duplication or data loss due to unstable `inode` values provided by the file system. Also, no support for network shares, containers or VMs. |
+| fingerprint | Files > 64 bytes in size (1 KB is a recommended default). Log files with unique content. | The most stable. Support for network shares, containers and VMs. | Slightly higher CPU / memory usage, does not start to ingest files before they reach the required size (1 KB by default). |
+
+### `path`
+
+This is the most basic file identity that uses file paths as their identifiers.
+
+If you're completely certain that your environment does not have log rotation, does not ever rename or move files, then you might consider using paths as unique stable identifiers for each file. This is the simplest but the riskiest option and it's not recommended to use.
+
+### `native`
+
+This file identity implementation is using:
+
+* For Unix-like systems: the `device` number and [`inode`](https://www.man7.org/linux/man-pages/man7/inode.7.html) from the [`stat`](https://www.man7.org/linux/man-pages/man2/stat.2.html) call. Can be viewed by using the `stat` command on any file.
+* For Windows: `dwVolumeSerialNumber`, `nFileIndexLow` and `nFileIndexHigh` values from the [file handler](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/ns-fileapi-by_handle_file_information).
+
+It combines these values together into a unique file identifier.
+
+It's stable on most common file systems. However, when used on some uncommon file systems, network shares, in containers or in virtualized environments it might lead to data duplication. This happens due to the re-use of `inode` values by the file system or a change of device numbers after reboot or reconnected network.
```

Comment:
```suggestion
It's stable under normal circumstances. However, when used in some environments with uncommon file systems, on network shares, in containers or in virtualized environments it might lead to data duplication. This happens due to the re-use of `inode` values by the file system or a change of device numbers after reboot or reconnected network.
```